lib/Varien/Object.php

```diff
@@ -854,4 +854,21 @@ public function flagDirty($field, $flag = true)
         }
         return $this;
     }
+
+    /**
+     * @throw Exception
+     * return string
+     */
+    public function __toString()
```

Comment:
I like the idea a lot, I was wondering, is it possible that somebody is already casting a varien_object to string for some reason (or mistake) and we're braking something?

Reply:
I suppose that if you cast to string any object that extends `Varien_Object`, you have your custom `__toString()` method? But I saw that PHPStan is unhappy.

Reply:
very weird, doesn't seem to be related to this PR...